docs/changelog-fragments/597.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix log level mapping to cover whole libssh range -- by :user:`Jakuje`
```

Comment:
This would need to have some context for the end-users, past tense (or non-imperative present simple with the reference to previous release) and a trailing period for as long as the byline is within the sentence.